app/controllers/pieces_controller.rb

```diff
@@ -27,4 +26,4 @@ def piece_params
     params.require(:piece).permit(:piece_type, :piece_color, :current_position_x, 
         :current_position_y, :starting_position_x, :starting_position_y)
```

Comment:
Good catch. They were in the db originally, but were removed here: `/db/migrate/20170712183347_remove_starting_positions_from_piece.rb` I have added this as an issue in the backlog.